packages/elements/src/components/ui/action-menu/ActionMenuLink.tsx

```diff
@@ -99,15 +99,15 @@ export const ActionMenuLink: React.FC<ActionMenuLinkProps> = ({
     >
       <Row
         height={theme.itemHeight}
-        indent
+        indent={2}
         alignItems={"center"}
         justifyContent={"space-between"}
         background={colors.itemBackground}
       >
         <Row height={theme.itemHeight} alignItems={"center"}>
           {icon && (
             <>
-              <Icon icon={icon} size={16} color={colors.iconColor} />
+              <Icon icon={icon} fixedWidth size={16} color={colors.iconColor} />
```

Comment:
We rarely want fixed width on icons since the heigh/width ratio can vary.

Setting fixed width will result in one of two things:
1) The icon gets stretched.
2) The height is affected.

And neither is desirable.

So the solution is to wrap the icon with a fixed-width container that centers the icon horizontally. 

For example.

```
<Row justifyContent={"center"} width={"32px"}>
   <Icon ... />
</Row>
```

So the width of the icon has some wiggle room and shouldn't affect the width of the parent.

In some cases, it might be OK that the height changes, but I don't think that is the case here.

Reply:
This is what setting fixedWidth does: https://github.com/FortAwesome/Font-Awesome/blob/master/scss/_fixed-width.scss

```scss
.#{$fa-css-prefix}-fw {
  text-align: center;
  width: $fa-fw-width;
}
```

It sets the width and centers the icon. Why the extra Row?

Reply:
My experience with fixedWidth in FontAwesome is what lindskogen describes. The whole point is to make all icons have the same "container width" without affecting the size of the icon. Have we broken this behaviour in web ui?

Reply:
Yay, nice! 🥳